docs/core-concepts/atomic-swap-htlc.md

```diff
@@ -0,0 +1,146 @@
+---
+id: atomic-swap-htlc
+title: Atomic Swaps using HTLCs
+sidebar_label: Atomic Swaps using HTLCs
+---
+
+This documentation section provides an introduction to atomic swaps, hash time locked contracts (HTLCs) and how these concepts are used in COMIT.
+Atomic Swaps and HTLCs are core concepts of the COMIT protocol.
+
+<iframe 
+    src="https://docs.google.com/presentation/d/e/2PACX-1vRjJeto_eQaZ8k5Ct00UzceHaSs-uIuvPmTFV6jP9SP6N-EKmct8H4cfN3xa2h-RFs8Gm4Gia41Je7m/embed?start=true&loop=false&delayms=1000" 
+    frameborder="0" 
+    width="801" 
+    height="480" 
+    allowfullscreen="true"
+    mozallowfullscreen="true" 
+    webkitallowfullscreen="true">
+</iframe>
+
+This slide shows an overview of how atomics swaps using HTLCs work in COMIT, more details can be found in the sections below.
+
+## Atomic swap
+
+An Atomic Swap is an exchange of two digital assets between two parties in which both parties are guaranteed to either receive the other asset OR retain their original asset i.e., an atomic swap guarantees that neither party is able to take both assets.
+
+Atomicity of the swap can be achieved by [HTLC](#hash-time-locked-contract-htlc)s that serve as a programmable escrow, or other concepts as explained in the [Bitcoin Wiki](https://en.bitcoin.it/wiki/Atomic_swap).
```

Comment:
I don't think we should tell people to go elsewhere for an explanation. At the very least, what do you think of putting some of the explanation here?

Reply:
Since `core-concepts` is about explaining things in a more general way, I would actually say we **should** link to other documentation. The Bitcoin Wiki is written in a nice way and I see no value in rephrasing that. I don't think the link hurts us, on the contrary. (The Bitcoin Wiki is also cited by other documentation like https://liquality.io/blog/hash-time-locked-contracts-htlcs-explained/) - other opinions?

Reply:
One alternative is to add a brief snippet of the most important part from the wiki.